JPS_Ontology/ontology/ontoems/OntoEMS.owl

```diff
@@ -998,8 +998,8 @@
 
     <owl:Class rdf:about="https://www.theworldavatar.com/kg/ontoems/PM2.5Concentration">
         <rdfs:subClassOf rdf:resource="https://www.theworldavatar.com/kg/ontoems/ParticulateMatterConcentration"/>
-        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">The concentration of particulate matter &lt; 2.5 µm in the ambient air.</rdfs:comment>
-        <rdfs:isDefinedBy rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://www.theworldavatar.com/kg/ontoems/</rdfs:isDefinedBy>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">The concentration of particulate matter &lt; 2.5 Âµm in the ambient air.</rdfs:comment>
```

Comment:
Oh no, I did not noticed that. Thanks for pointing it out. I think it will be faster if I manually fix them.

Reply:
> Oh no, I did not noticed that. Thanks for pointing it out. I think it will be faster if I manually fix them.

Sounds good. Although I am not convinced that all equivalent classes are added back by now, e.g. `PrecipitationProbability` seems still missing (and potentially more). Please have a double check before re-requesting final review. Thanks

Reply:
Oh but I compared the edited file with the one in main and there's no equivalent classes for PrecipitationProbability or any others. Let me compare with the csv file. They were probably not added in when the owl file was last modified.

Reply:
I have added the missing equivalent classes definitions which were also missing from main. I compared the csv file with the owl file and all equivalent classes definitions should be accounted for.